### PR TITLE
Add ao vpn ports

### PR DESCRIPTION
**Pull request by patrick-fox-cisa**
## 🗣 Description ##

This PR makes 100 additional TCP and 100 additional UDP ports available for use by the AO team.

## 💭 Motivation and context ##

The current number of ports is no longer enough to meet the needs of the AO team.

## 🧪 Testing ##

- [x] Applied this change in Staging
- [x] @patrick-fox-cisa verified that the ports are accessible in Staging as expected

## ✅ Pre-approval checklist ##

- [x] This PR has an informative and human-readable title.
- [x] Changes are limited to a single goal - *eschew scope creep!*
- [x] All relevant type-of-change labels have been added.
- [x] I have read the [CONTRIBUTING](../blob/develop/CONTRIBUTING.md) document.
- [x] These code changes follow [cisagov code standards](https://github.com/cisagov/development-guide).
- [x] All new and existing tests pass.

## ✅ Post-merge checklist ##

- [x] Apply this change in Production

### CODE REVIEW

File: locals.tf [hunk 1]
```diff
@@ -45,6 +45,16 @@ locals {
...
+    ao_vpn_endpoints_2_tcp = {
+      from_port = 60000
+      protocol  = "tcp"
+      to_port   = 60100
+    },
+    ao_vpn_endpoints_2_udp = {
+      from_port = 60000
+      protocol  = "udp"
+      to_port   = 60100
+    },
```
Comment (dav3r) [inline locals.tf:57]: What about simply expanding the current set of port numbers, i.e. something like this?

```hcl
    ao_vpn_endpoints_tcp = {
      from_port = 51820
      protocol  = "tcp"
      to_port   = 51935
    },
    ao_vpn_endpoints_udp = {
      from_port = 51820
      protocol  = "udp"
      to_port   = 51935
    },
```

Comment (patrick-fox-cisa) [reply locals.tf:57]: I picked `60000-60100` arbitrarily just to have easier numbers to work with our applications. So `-51935` is possible but less desirable

Comment (dav3r) [reply locals.tf:57]: Do you still want to keep the original 51820-51835 UDP range?  If not, then we can go with this instead:

```hcl
    ao_vpn_endpoints_tcp = {
      from_port = 60000
      protocol  = "tcp"
      to_port   = 60100
    },
    ao_vpn_endpoints_udp = {
      from_port = 60000
      protocol  = "udp"
      to_port   = 60100
    },
```

Comment (patrick-fox-cisa) [reply locals.tf:57]: Yes, let's keep those - I don't want to break existing functionality at the moment.

Comment (dav3r) [reply locals.tf:57]: Sounds good.  Can you please update your change to look like this (preserves alphabetical order for the keys and renames them to align with how we would do it)?

```hcl
    ao_vpn_endpoints_tcp = {
      from_port = 60000
      protocol  = "tcp"
      to_port   = 60100
    },
    ao_vpn_endpoints_udp_1 = {
      from_port = 51820
      protocol  = "udp"
      to_port   = 51835
    },
    ao_vpn_endpoints_udp_2 = {
      from_port = 60000
      protocol  = "udp"
      to_port   = 60100
    },

Comment (dav3r) [reply locals.tf:57]: https://github.com/cisagov/cool-sharedservices-openvpn/pull/76/commits/5f6a9b13612303f2fd73e468c6f1a6a19bb450da looks good. 👍

**Comment by dav3r** [comment]
@patrick-fox-cisa I applied this change in Staging.  Please verify that it works as expected.

**Comment by patrick-fox-cisa** [comment]
Tested and working as expected, including the old ports.

**Review by jsf9k — APPROVED** [review]

**Review by dav3r — APPROVED** [review]
Thanks for contributing @patrick-fox-cisa!

**Comment by dav3r** [comment]
@patrick-fox-cisa - I just applied this change in Production as well.

**Pull request merged**